### DISCUSSION TITLE
该议题无效内容自行诊断

### DISCUSSION THREAD

**Discussion by weekstarR2S** [AlistGo/alist#3334] [Invalid question]
该议题无效内容自行诊断

**Comment by weekstarR2S** [comment]


**Reply by dreamsamethyst** [reply]
不能啊，我看官网都把阿里云盘给去掉了呀

**Reply by dreamsamethyst** [reply]
没，一会我再去试试

**Reply by dreamsamethyst** [reply]
获取不到token

**Reply by dreamsamethyst** [reply]
刚才获取到token了，但是依然不行 ，视频无法播放，显示重试，最后失败
![Uploading 6.png…]()

**Reply by dreamsamethyst** [reply]
![6](https://user-images.githubusercontent.com/106142275/218996988-e58a0574-476d-40a4-b067-6270f94c5caf.png)

**Reply by Dashan-IZ** [reply]
> ![6](https://user-images.githubusercontent.com/106142275/218996988-e58a0574-476d-40a4-b067-6270f94c5caf.png)
> 
> 

你适合进qq群

**Reply by dreamsamethyst** [reply]
好嘞，大佬

**Comment by Dashan-IZ** [comment]
这是开发者大佬自己的互助qq群：746260284

**Reply by Dashan-IZ** [reply]
![3477ba18b13f36dec6d4b9a0b2a3965](https://user-images.githubusercontent.com/78125220/219327831-72ef9ecd-60ed-4cc5-9ddc-2ab2a48d98c7.jpg)
![f37d344ab37b16006c6b9fa3938bcad](https://user-images.githubusercontent.com/78125220/219327878-0a2099f4-3356-4601-9bb3-83dcf1852bb5.jpg)
![bcd2b852f3eb2254ec9c87aec5f51db](https://user-images.githubusercontent.com/78125220/219328128-108b14e4-1927-458b-b20f-ef43d1cb61eb.jpg)

别人自愿的昂，本群不收费，除非你忍不住要给钱，我也不好意思拒绝，我帮别人用todesk远程搞了挺久，别人说有偿，还是他先提出来的，我才要了瓶红牛有什么错，搞不懂了，就

**Reply by Dashan-IZ** [reply]
本群不收费的昂，大佬多多，都平易近人，有问必答：qq群：746260284

**Comment by txj1996** [comment]
请问这个问题怎么办？failed get objs: failed to list objs: ocr error:

**Comment by GSANDLY** [comment]
由于阿里云盘 referer 的限制，必须使用移动端 token。 使用桌面 Web 令牌将导致无法下载和预览。 当然，如果你在本地使用或者带宽足够大，你也可以开启代理，让桌面 Web 的 refresh token 正常工作。

https://alist.nn.ci/zh/guide/drivers/aliyundrive.html
